keps/sig-node/1029-ephemeral-storage-quotas/README.md

```diff
@@ -840,65 +890,78 @@ If LocalStorageCapacityIsolationFSQuotaMonitoring is turned on but LocalStorageC
 
 ### Monitoring Requirements
 
-* **How can an operator determine if the feature is in use by workloads?**
+###### How can an operator determine if the feature is in use by workloads?
 
   - In kubelet metrics, an operator can check the histgram metric `kubelet_volume_metric_collection_duration_seconds`
     with metric_source equals "fsquota". If there is no `metric_source=fsquota`, this feature should be disabled.
   - However, to figure out if a workload is use this feature, there is no direct way now and see more in below 
     methods of how to check fsquota settings on a node.
 
-* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+###### What are the reasonable SLOs (Service Level Objectives) for the above SLIs?
 
-  - 99.9% of volume stats calculation will cost less than 1s or even 500ms.
-  It can be calculated by `kubelet_volume_metric_collection_duration_seconds` metrics.
+99.9% of volume stats calculation will cost less than 1s or even 500ms.
+It can be calculated by `kubelet_volume_metric_collection_duration_seconds` metrics.
 
-* **What are the SLIs (Service Level Indicators) an operator can use to determine
-the health of the service?**
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
 - [x] Metrics
   - Metric name: `kubelet_volume_metric_collection_duration_seconds`
   - Aggregation method: histogram
   - Components exposing the metric: kubelet
 
-* **Are there any missing metrics that would be useful to have to improve observability of this feature? **
+###### Are there any missing metrics that would be useful to have to improve observability of this feature? **
 
-  - Yes, there are no histogram metrics for each volume. The above metric was grouped by volume types because
-    the cost for every volume is too expensive. As a result, users cannot figure out if the feature is used by
-    a workload directly by the metrics. A cluster-admin can check kubelet configuration on each node. If the
-    feature gate is disabled, workloads on that node will not use it. 
-    For example, run `xfs_quota -x -c 'report -h' /dev/sdc` to check quota settings in the device. 
-    Check `spec.containers[].resources.limits.ephemeral-storage` of each container to compare.
+Yes, there are no histogram metrics for each volume. The above metric was grouped by volume types because
+the cost for every volume is too expensive. As a result, users cannot figure out if the feature is used by
+a workload directly by the metrics. A cluster-admin can check kubelet configuration on each node. If the
+feature gate is disabled, workloads on that node will not use it. 
+For example, run `xfs_quota -x -c 'report -h' /dev/sdc` to check quota settings in the device. 
+Check `spec.containers[].resources.limits.ephemeral-storage` of each container to compare.
 
+###### How can someone using this feature know that it is working for their instance?
```

Comment:
Nit, this question should be a bit higher in the doc, right after `###### How can an operator determine if the feature is in use by workloads?` :wink: